apm-collector/apm-collector-boot/src/main/resources/application.yml

```diff
@@ -31,3 +31,9 @@ storage:
     cluster_nodes: localhost:9300
     index_shards_number: 2
     index_replicas_number: 0
+# uncomment to enable h2 storage
+#storage:
+#    h2:
+#      url: jdbc:h2:~/collector
+#      user_name: sa
+#      password: sa
```

Comment:
@peng-yongsheng @clevertension  I prefer use h2 as default storage, in order to start up collector easier. What do you think? And I recommend don't use zookeeper as default. By these two, our collector quick start is pretty easy. 

```yml
# Uncomment to make collector running in cluster mode, by zookeeper as a coordinator.
# cluster:
#   zookeeper:
#     hostPort: localhost:2181
#     sessionTimeout: 100000
```

Reply:
two thumbs up， i can't agree more!
we need to configure easy, run as quickly as possible to show the demos to users

Reply:
+1